test/blackbox-tests/test-cases/variants-external-declaration/run.t

```diff
@@ -24,4 +24,4 @@ declared in the virtual library definition.
       (visibility public)
       (kind virtual)
       (intf)))
-    (wrapped true)))
+    (wrapped false)))
```

Comment:
Are that changes and the next one expected?

Reply:
Currently, it's a bit ambiguous what to set `wrapped` for libraries with only 1 module. I'll look into changing this.